docs/optimization_models.rst

```diff
@@ -2,4 +2,619 @@
+
+.. _edges_csv :
+
+Pipes
+'''''
+
+.. csv-table::
+   :header-rows: 1
+   :file: _static/opti_edges.csv
+   :align: center
+
+The following optional attributes must be given in every *ThermalNetwork*:
+
+* **id**: see :ref:`Thermal Network <thermal_network_label>`
+* **from_node**: see :ref:`Thermal Network <thermal_network_label>`
+* **to_node**: see :ref:`Thermal Network <thermal_network_label>`
+* **length**: see :ref:`Thermal Network <thermal_network_label>`
+
+The following optional attributes are introduced by the optimization module:
+
+* **existing**: Binary indicating an existing pipe. If there is no column
+  *existing* given, all Edges are free for optimization.
+* **capacity**: Capacity of existing pipes.
+  If *existing* is *True*, a *capacity* must be given.
+* **hp_type**: Label of the type of pipe. The *hp_type* refers to
+  a set of parameters of a pipeline component. The parameters for the
+  *hp_type* must be given in the following table (see :ref:`network/pipes.csv <invest_pipes>`).
+  If *existing* is *True*, a *hp_type* must be given.
+* **active**: Binary indicating that this edge is considered. If no column
+  *active* is given, all edges are active. With this attribute, single edges
+  can be switched on and off. This can be very useful, if different scenarios
+  should be analyzed, e.g. you might like to make a given street/edges unavailable.
+* **add_fix_costs**: (*not implemented yet*) Additional fix investment costs. Depending on the
+  street and route of a DHS, the construction costs might differ. With this parameter,
+  additional fix investment costs (independent of the size of the pipes) can be
+  considered for each edge individually. Therefore, you need to apply a
+  nonconvex investment (see :ref:`network/pipes.csv <invest_pipes>`).
+
+Consumers
+'''''''''
+
+.. csv-table::
+   :header-rows: 1
+   :file: _static/opti_consumers.csv
+   :align: center
+
+The following optional attributes must be given in every *ThermalNetwork*:
+
+* **id**: see :ref:`Thermal Network <thermal_network_label>`
+
+The following optional attributes are introduced by the optimization module:
+
+* **active**: Binary indicating that consumer-xy is considered. If no column
+  *active* is given, all consumers are active. With this attribute, single
+  consumers can be switched on and off (e.g. for scenario analysis with
+  different connection quotes).
+* **P_heat_max**: Maximum heat load of consumer. If no column
+  *P_heat_max* is given, the maximum heat load is calculated from the heat
+  demand series (see `consumers-heat_flow.csv`). Depending on the optimization
+  setting, *P_heat_max* or the demand series is used for the optimization (see
+  :ref:`Optimization settings` for further information).
+
+Producers
+'''''''''
+
+.. csv-table::
+   :header-rows: 1
+   :file: _static/opti_producers.csv
+   :align: center
+
+The following optional attributes must be given in every *ThermalNetwork*:
```

Comment:
I would rather not repeat the attributes of the ThermalNetwork. Repeated information means extra maintenance work. Just show the extra attibutes required for optimization models. In the beginning of the section, this could be clarified: Here we only present the optimizatoin specific attributes.

Reply:
Yes, you are right. I cut the tables to the additional attributes of the optimisation approach ...